MANIFEST.in

```diff
@@ -1,4 +1,4 @@
 recursive-include tests *.py
 recursive-include immutables *.py *.c *.h *.pyi
 include LICENSE README.rst
-include immutables/py.typed
+include mypy.ini immutables/py.typed
```

Comment:
What's the reason for adding `mypy.ini` in the distribution?

Reply:
I think I did this out of habit. Most of the projects I work on do some sort of type checking during testing. That doesn't seem to be the case for this one.

Reply:
Actually, I think it is good idea to add such a test, otherwise the annotations will inevitably rot.

Reply:
@elprans would you like me to add a test that runs mypy?

Reply:
Sure, that would work too.